Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+
+    response = client.retrieve_a_specific_incident_details_request(lumu_incident_id)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveASpecificIncidentDetails",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_a_specific_incident_context_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    hash = args.get("hash")
+
+    response = client.retrieve_a_specific_incident_context_request(
+        lumu_incident_id, hash
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveASpecificIncidentContext",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def comment_a_specific_incident_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    comment = generate_hmac_sha256_msg(
+        client.api_key, add_prefix_to_comment(args.get("comment"))
+    )
+    try:
+
+        response = client.comment_a_specific_incident_request(lumu_incident_id, comment)
+    except DemistoException as err:
+
+        error_msg = "Failed to parse json object from response: b''"
+        if str(err) == error_msg:
+            response = {"statusCode": 200}
+        else:
+            raise DemistoException(err)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.CommentASpecificIncident",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_open_incidents_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+    adversary_types = args.get("adversary-types")
+    labels = args.get("labels")
+
+    if adversary_types and isinstance(adversary_types, str):
+        adversary_types = adversary_types.split(",")
+    if labels and isinstance(labels, str):
+        labels = [int(label) for label in labels.split(",")]
+
+    response = client.retrieve_open_incidents_request(
+        page, items, adversary_types, labels
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveOpenIncidents",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_muted_incidents_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+    adversary_types = args.get("adversary-types")
+    labels = args.get("labels")
+
+    if adversary_types and isinstance(adversary_types, str):
+        adversary_types = adversary_types.split(",")
```

Comment:
please use `argToList` in all the places for list arguments (like retrieve_incidents_command, retrieve_open_incidents_command, etc...) and update the yml file with `isArray: true` 
```suggestion
    adversary_types = argToList(adversary_types)
```

Reply:
replaced them in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7

Reply:
updated in recent commit